lib/utils/type.js

```diff
@@ -19,50 +19,63 @@ const contentType = require('content-type');
 
 const DEFAULT_MIME_TYPE = 'application/octet-stream';
 
-async function detectTypeUsingFileCommand(renditionPath) {
-    const result = {};
-
-    // assumes rendition exists and size > 0 (to be checked in caller)
-    try {
-        const { stdout } = await exec(`file -b --mime "${renditionPath}"`);
-
-        // parsing will throw if `file` command did not return a valid content type
-        const type = contentType.parse(stdout.trim());
-        result.mime = type.type || DEFAULT_MIME_TYPE;
-
-        if (type.parameters && type.parameters.charset) {
-            result.encoding = type.parameters.charset;
+class Mimetype {
```

Comment:
I would recommend not refactoring the type.js file for this necessary fix but instead do it in a follow up PR

Reply:
the main reason being, we export `detectContentType` so it would be a considered a breaking change

Reply:
Works with me.